Testscripts/Linux/XDPDumpSetup.sh

```diff
@@ -110,6 +112,25 @@ function Run_XDPDump {
     fi
 }
 
+# Check if kernel supports XDP or not
+function kernel_check {
```

Comment:
I think this function can be supported as an utility function. Can we define this in utils.sh or other common file so other test scripts can use it ?

Reply:
For the current xdp testcases, this script is executing to make sure XDP is running on the vm. I don't see any other benefit of adding this to utils.sh or any other common file.